cgmanifest.json

```diff
@@ -175,8 +175,28 @@
         "type": "other",
         "other": {
           "name": "asciidoc",
-          "version": "8.6.10",
-          "downloadUrl": "https://github.com/asciidoc/asciidoc-py3/archive/8.6.10.tar.gz"
+          "version": "9.1.0",
+          "downloadUrl": "https://github.com/asciidoc-py/asciidoc-py/releases/download/9.1.0/asciidoc-9.1.0.tar.gz"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "aspnetcore-runtime-3.1",
+          "version": "3.1.5",
+          "downloadUrl": "https://download.visualstudio.microsoft.com/download/pr/6827d794-a218-4352-b3b3-a19ec773c975/e3e53bc2f20df220a29c6e09f74d8a00/aspnetcore-runtime-3.1.5-linux-x64.tar.gz"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "at",
+          "version": "3.1.23",
+          "downloadUrl": "http://ftp.debian.org/debian/pool/main/a/at/at_3.1.23.orig.tar.gz"
```

Comment:
It looks like this adds some duplicate entries to the cgmanifest- probably a bad auto merge, but can we remove this?

Reply:
Doing that: for reference in the future, I used the following jq command to find the duplicates

    jq -c '.Registrations[].component.other | [.name, .version]' cgmanifest.json | sort | uniq -d